gettsim/parameters/arbeitsl_geld_2.yaml

```diff
@@ -282,42 +284,77 @@ mehrbedarf_anteil:
     max: 0.6
     reference: G. v. 24.12.2003 BGBl. I S. 2954.
 
-vermögensfreibetrag:
+
+vermögensfreibetrag_obergrenze:
   name:
-    de_1: Vermögensgrundfreibetrag je Lebensjahr, nach Geburtsjahr, erst ab 2008 unterschiedlich
-    de_2: Vermögensfreibetrag für Erstausstattung
-    de_3: Vermögensgrundfreibetrag je Kind
-    en_1: Basic allowance per year of life, sorted by year of birth, starting 2008 different
-    en_2: Allowance for furniture
-    en_3: Allowance per child
+    de: Maximaler Vermögensgrundfreibetrag je Lebensjahr. Ab 2008 nach Geburtsjahr unterschiedlich
+    en: Basic allowance per year of life. Sorted by year of birth starting 2008 different
   description:
     de: § 12 (2) Satz 1 Nr. 1 SGB II.
     en:
   unit: Euro
   2005-01-01:
-    standard: 200
-    vor_1948: 520
-    1948_bis_1957: 13000
-    1958_bis_1963: 13000
-    nach_1963: 13000
-    ausstattung: 750
-    kind: 4100
+    scalar: 13000
```

Comment:
I would prefer having the same keys (`vor_1957`, etc.) over time in this case since it allows to define `max_freibetrag_vermögen_hh` in one function and therefore reduces complexity. I don't think it is problematic if all keys have the same value before 2008.